src/server/capacity_unit_calculator.cpp

```diff
@@ -144,25 +161,28 @@ void capacity_unit_calculator::add_scan_cu(int32_t status,
     int64_t data_size = 0;
     for (const auto &kv : kvs) {
         data_size += kv.key.size() + kv.value.size();
+        _read_hotkey_collector->capture_raw_key(kv.key, 1);
     }
     add_read_cu(data_size);
     _pfc_scan_bytes->add(data_size);
 }
 
-void capacity_unit_calculator::add_sortkey_count_cu(int32_t status)
+void capacity_unit_calculator::add_sortkey_count_cu(int32_t status, const dsn::blob &hash_key)
 {
     if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound) {
         return;
     }
     add_read_cu(1);
+    _read_hotkey_collector->capture_hash_key(hash_key, 1);
 }
 
-void capacity_unit_calculator::add_ttl_cu(int32_t status)
+void capacity_unit_calculator::add_ttl_cu(int32_t status, const dsn::blob &key)
 {
     if (status != rocksdb::Status::kOk && status != rocksdb::Status::kNotFound) {
         return;
     }
     add_read_cu(1);
+    _read_hotkey_collector->capture_raw_key(key, 1);
```

Comment:
refactor to move L184~L185 to a function

Reply:
`add_read_cu(1);` and `_read_hotkey_collector->capture_raw_key(key, 1);` have different semantics, they are not suitable for merging
@zhangyifan27

Reply:
I mean you should wrap these two lines of code into a new function, but not merge `_read_hotkey_collector->capture_raw_key(key, 1);` into `add_read_cu(1)`.
Because these two lines are always called in the same time.

Reply:
If these two functions are always called at the same time, you can put them into a new function `collect_hotkey_and_cu`